.readthedocs.yml

```diff
@@ -11,6 +11,7 @@ build:
       - poetry config virtualenvs.create false
     post_install:
       - poetry install --with docs
+      working-directory: packages/service
```

Comment:
GitHub and ReadTheDocs have separate YAML schemas. I see no reason to expect this to work. RTD uses underscores not dashes, and I don't see this key documented in https://docs.readthedocs.io/en/stable/config-file/v2.html

I would try `poetry -C packages/service/ install --with docs` or `pushd packages/service/ && poetry install --with docs`.

To test it, go to https://readthedocs.org/projects/measurement-plugin-python/versions/ and activate your dev branch.

Reply:
poetry -C seems to work. Now the error is that Sphinx can't find `autoapi`.

It looks like RTD's Poetry instructions have changed since I set this up: https://docs.readthedocs.io/en/stable/build-customization.html#install-dependencies-with-poetry

Reply:
Looks like it worked: https://measurement-plugin-python.readthedocs.io/en/users-jdixon-fix-post-install-readthedocs/index.html